css-layout-classes/styles.css

```diff
@@ -0,0 +1,26 @@
+.container {
+  max-width: 800px;
+  margin-left: auto;
+  margin-right: auto;
+  display: flex;
+  flex-wrap: wrap;
```

Comment:
`flex` is actually not needed on the `container` if you are using a `max-width` and `margin: auto` to center the content of the entire page